pkg/checkers/oom/oom.go

```diff
@@ -33,9 +34,18 @@ func (c *OOMChecker) Name() string {
 }
 
 func New() *OOMChecker {
-	//todo: support other logpath
+	paths := []string{kmsgLogPath, ubuntuLogPath}
+	for _, path := range paths {
+		file, err := os.Open(path)
+		file.Close()
+		if err != nil {
+			return &OOMChecker{
```

Comment:
Looks confusing. How about we return if it passes the check?

```
if file, err := os.Open(path); err == nil {
    file.Close()
    return &OOMChecker{kernLogPath: path}
}
```

Reply:
Fixed.